presto-main/src/test/java/com/facebook/presto/operator/scalar/FunctionAssertions.java

```diff
@@ -680,7 +690,8 @@ public ConnectorPageSource createPageSource(Session session, Split split, List<C
                         new DateTime(2001, 8, 22, 3, 4, 5, 321, DateTimeZone.UTC).getMillis(),
                         "%el%",
                         null,
-                        packDateTimeWithZone(new DateTime(1970, 1, 1, 0, 1, 0, 999, DateTimeZone.UTC).getMillis(), TimeZoneKey.getTimeZoneKey("Z"))
+                        packDateTimeWithZone(new DateTime(1970, 1, 1, 0, 1, 0, 999, DateTimeZone.UTC).getMillis(), TimeZoneKey.getTimeZoneKey("Z")),
+                        new BinaryLiteral("0a 0b 0c").getSlice().getBytes()
```

Comment:
for this line, the Slices.wrappedBuffer().getBytes() approach might be simpler and faster, but I thought explicitly creating a BinaryLiteral may be clearer to the reader that the test is for binaryLiteral. since this is just unit tests, the small difference in efficiency doesn't seem that important to me.